test/Microsoft.NET.TestFramework/RepoInfo.cs

```diff
@@ -102,5 +102,15 @@ public static string GetBaseDirectory()
 
             return directory;
         }
+
+        public static ICommand AddTestEnvironmentVariables(ICommand command)
+        {
+            //  Set NUGET_PACKAGES environment variable to match value from build.ps1
+            command = command.EnvironmentVariable("NUGET_PACKAGES", Path.Combine(RepoInfo.RepoRoot, "packages"));
```

Comment:
Yeah, that wouldn't hurt.  This code was written before that variable existed, and I'm moving it to a different location here.